hedera-node/hedera-evm-api/src/main/java/module-info.java

```diff
@@ -2,15 +2,16 @@
 module com.hedera.services.evm {
     requires org.hyperledger.besu.evm;
     requires org.hyperledger.besu.datatypes;
-    requires com.google.protobuf;
     requires com.hedera.hashgraph.protobuf.java.api;
-    requires org.apache.commons.lang3;
     requires tuweni.bytes;
     requires javax.inject;
     requires com.swirlds.common;
+    requires tuweni.units;
```

Comment:
we should not include units if we can.  Is this the same HederaBlockValues issue as seen in another pr?